qubes/tools/qvm_create.py

```diff
@@ -42,15 +42,21 @@
     default='AppVM',
     help='specify the class of the new domain (default: %(default)s)')
 
-parser.add_argument('--property', '--prop', '-p',
+parser.add_argument('--property', '--prop',
```

Comment:
I object to this change. I think `--property` would be used more often than `--pool`.